src/full/Agda/Main.hs

```diff
@@ -44,15 +44,9 @@ import qualified Agda.Utils.Benchmark as UtilsBench
 import Agda.Utils.Except ( MonadError(catchError, throwError) )
 import Agda.Utils.Impossible
 
-builtinBackends :: [Backend]
-builtinBackends = [ ghcBackend, jsBackend ]
-
 -- | The main function
 runAgda :: [Backend] -> IO ()
-runAgda backends = runAgda' $ builtinBackends ++ backends
-
-runAgda' :: [Backend] -> IO ()
-runAgda' backends = runTCMPrettyErrors $ do
+runAgda backends = runTCMPrettyErrors $ do
```

Comment:
Is there a good reason for this change? It will break custom backends who will now have to explicitly include the builtin backends.

Reply:
Hum, IMO, any particular backend is logically independent from this module. In this way any change to built-in backends will not require GHC (or GHCi) to reload `Agda.Main`. 

But, as you pointed out that it is a breaking change, it may do more harm than good.

Reply:
For instance, this is the main function of the [ocaml backend](https://github.com/agda/agda-ocaml/blob/e38b699870ba638221828b07b12948d70a1bdaec/src/full/Agda/Compiler/Malfunction/Main.hs#L9):
```haskell
-- | Invokes the agda-compiler with the additional malfunction backend.
main :: IO ()
main = runAgda [Malfunction.backend]
```
With your change this would break, dropping the builtin backends.

Reply:
Okay, thanks. I will revert this part and put `runAgda` and `runAgda’` back.